client/signup/style.scss

```diff
@@ -307,27 +307,6 @@ body.is-section-signup .layout:not(.dops):not(.is-wccom-oauth-flow) {
 
 	.signup__step.is-user,
 	.signup__step.is-user-social {
-		.button.signup-form__submit[disabled],
-		.button.signup-form__submit:disabled,
-		.button.signup-form__submit.disabled {
-			background-color: var(--studio-gray-20);
-			color: var(--color-surface);
-		}
-
-		.button.signup-form__submit {
-			border-radius: 4px;
-			max-width: 408px;
-			height: 44px;
-			box-shadow: 0 1px 2px rgba(0, 0, 0, 0.05);
-			font-weight: 500;
-			letter-spacing: 0.32px;
-			line-height: 17px;
-
-			&.newsletter-signup-form {
-				background-color: var(--color-primary);
-			}
-		}
```

Comment:
This was the last `signup-form__submit` button to be migrated to a core button so these styles are no longer needed.